### DISCUSSION TITLE
多个微应用使用相同的主要依赖，加载资源大小会加倍吗？

### DISCUSSION THREAD

**Discussion by WangShuXian6** [jd-opensource/micro-app#241] [Q&A]
>例如多个相同版本的react项目组成的一个大应用，加载时资源包也会重复加载吗？

>大型管理后台项目每个一级菜单作为一个微前端应用，那么首次加载速度会稍慢吗？

>这样是不是就像小程序相互跳转一样，跳转过度需要等待全部资源加载，导致应用之间的切换时间会比非微前端页面切换慢一些？

**Comment by bailicangdu** [comment]
参考文档[资源共享](https://micro-zoe.github.io/micro-app/docs.html#/zh-cn/static-source?id=%e8%b5%84%e6%ba%90%e5%85%b1%e4%ba%ab)

**Reply by bailicangdu** [reply]
相同的js或css资源可以指定为全局资源，所有子应用可以复用同一份资源，不需要重复下载

**Comment by bailicangdu** [comment]
对于非首屏微应用可以使用预加载，提升初次渲染速度
